Formula/pgrok.rb

```diff
@@ -25,20 +25,16 @@ def install
       -X main.date=#{time.iso8601}
     ]
 
-    ["pgrokd", "pgrok"].each do |f|
+    ["pgrok"].each do |f|
       system "go", "build", *std_go_args(ldflags: ldflags, output: bin/f), "./cmd/#{f}"
     end
```

Comment:
just do this is good enough:
  system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/pgrok"